### DISCUSSION TITLE
Ansible Galaxy - Get a namespace for paperless-ngx to be able to publish role releases there

### DISCUSSION THREAD

**Discussion by stevenengland** [paperless-ngx/ansible#25] [General]
Hi together,

it would be great to publish role releases as Ansible Galaxy releases too so that they can be pulled by users via the Ansible ecosystem. I did that in the past with my role. See as an example: https://galaxy.ansible.com/stevenengland/paperless_ngx
Now I would like to publish them like so: https://galaxy.ansible.com/paperless-ngx/paperless_ngx

TLDR
Let's clarify some terms here: That role is published under the namespace "stevenengland" with the role name "paperless-ngx". The namespace is is tightly coupled to my account name on GitHub. As a standard I am only able to publish roles belonging to that namespace/roles that can be found under my GitHub account. To publish roles that live within another GitHub account (let's call it "orga1") that account must have an Ansible Galaxy account ("orga1)" as well OR an dedicated and manually (by the Galaxy staff) created namespace and the orga must have admistrated me in Ansible Galaxy as an admin.
Problem 1: There is no account/namespace paperless-ngx in Ansible Galaxy.
Problem 2: I am not an admin for the non existing namespace in Galaxy :)

What to do?
I created an issue at Ansible Galaxy so that the namespace paperless-ngx will be created. Hopefully. Thats a normal workflow to do so. You can read about it here: https://galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespace-requests
The issue can be found here: https://github.com/ansible/galaxy/issues/3118

More to discuss: 
If everything works as I intended to then I will become the owner of that namespace because until now I am the only one that will get administrated. If someone else also wants to become an admin for that namespace:
* Create an account at https://galaxy.ansible.com
* Open the issue and write a comment or tell me so that I can edit mine in order to get more admin accounts.

Is this okay so far?

Kind regards!

@shamoon @SiM22

**Comment by shamoon** [comment]
Sounds good, I could also create a username paperless-ngx over at galaxy but it sounds like you already started that namespace process so we can always add other folks as admins after it’s setup. That will be good to have some redundancy.

**Comment by stevenengland** [comment]
Namespace was created, shamoon, me and the paperless bot are admins right now ;) Therefore I will close this discussion.
